README-zh.md

```diff
@@ -70,32 +69,42 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
   - 不支持内核增强的OS版本：
 
     为了兼容不同的OS版本，Kmesh提供在线编译并运行的镜像。在Kmesh被部署之后，它会基于宿主机的内核能力自动选择运行的Kmesh特性，从而满足一个镜像在不同OS环境运行的要求。
-
-  ```
+    
+  - 针对所有OS版本：
+  
+    考虑到kmesh使用的通用性，我们发布了用于kmesh编译构建的镜像。用户可以基于此镜像方便的制作出可以在当前OS版本上运行的kmesh镜像。
+  
+  ```bash
   # 用于openEuler 23.03 OS的Kmesh x86镜像
-  docker pull ghcr.io/kmesh-net/kmesh:v0.1.0
-
-  # x86镜像用于Kmesh的在线编译以及执行，支持的OS版本为5.10及以上
-  docker pull ghcr.io/kmesh-net/kmesh-x86:v0.1.0
-
-  # The arm image for Kmesh online compilation and execution, supports OS kernel versions 5.10 and above.
-  # arm镜像用于Kmesh的在线编译以及执行，支持的OS版本为5.10及以上
-  docker pull ghcr.io/kmesh-net/kmesh-arm:v0.1.0
+  docker pull ghcr.io/kmesh-net/kmesh:v0.2.0
+  
+  # 在线编译镜像用于Kmesh的在线编译以及执行，支持的OS版本为5.10及以上
+  docker pull ghcr.io/kmesh-net/kmesh-compatible:v0.2.0
+  
+  # kmesh编译环境镜像
+  docker pull ghcr.io/kmesh-net/kmesh-build-x86:v0.2.0

```

Comment:
v0.2.0 -> leatest

Reply:
latest

Reply:
We donot need to pull explicitly, i think when we run make docker/make build, it will be pulled automatically